bl/errorReporter/errorReporter.go

```diff
@@ -0,0 +1,70 @@
+package errorReporter
+
+import (
+	"fmt"
+	"runtime/debug"
+
+	"github.com/datreeio/datree/cmd"
+	"github.com/datreeio/datree/pkg/cliClient"
+	"github.com/datreeio/datree/pkg/deploymentConfig"
+	"github.com/datreeio/datree/pkg/localConfig"
+)
+
+func ReportCliError(panicErr interface{}) {
+	reporter := NewErrorReporter(cliClient.NewCliClient(deploymentConfig.URL), localConfig.NewLocalConfig())
+	reporter.ReportCliError(panicErr)
+}
+
+type LocalConfig interface {
+	GetLocalConfiguration() (*localConfig.ConfigContent, error)
+}
+
+type CliClient interface {
+	ReportCliError(reportCliErrorRequest cliClient.ReportCliErrorRequest) (StatusCode int, Error error)
+}
+
+type ErrorReporter struct {
+	config LocalConfig
+	client CliClient
+}
+
+func NewErrorReporter(client CliClient, localConfig LocalConfig) *ErrorReporter {
+	return &ErrorReporter{
+		client: client,
+		config: localConfig,
+	}
+}
+
+func (reporter *ErrorReporter) ReportCliError(panicErr interface{}) {
+	status, err := reporter.client.ReportCliError(cliClient.ReportCliErrorRequest{
+		CliId:        reporter.getCliId(),
```

Comment:
I think this needs to be `token` instead of cliId because we are already in the process of changing naming conventions in all backend requests